flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1168,6 +1168,22 @@ class IBlockDeviceAPITestsMixin(object):
     """
     this_node = None
 
+    def _assert_volume_size(self, volume, size):
+        """
+        Assert that size of given volume matches provided size value.
+
+        :param BlockDeviceVolume volume: Volume we are interested in.
+        :param int size: Expected size of input volume.
+        """
+        volume_size = volume.size
+        device_path = self.api.get_device_path(volume.blockdevice_id).path
+        device = device_path.encode("ascii")
+
+        command = [b"/bin/lsblk", b"-nb", b"-o", b"SIZE", device]
+        command_output = check_output(command).split(b'\n')[0]
+        device_size = int(command_output.strip().decode("ascii"))
+        self.assertEquals(volume_size, device_size)
```

Comment:
`TestCase` provides like a million aliases for all of its methods.  `assertEqual` is the preferred spelling of this one, though (and I think some of the alternate spellings are even deprecated).